examples/aws-nodejs/index.js

```diff
@@ -190,7 +190,7 @@ app.get('/s3/multipart/:uploadId/:partNumber', (req, res, next) => {
   }, next)
 })
 
-app.get('/s3/multipart/:uploadId', (req, res, next) => {
+app.get('/s3/multipart/:uploadId', async (req, res) => {
```

Comment:
Using async/await in such handler is an anti-pattern, if there's an error, the client won't be notified – and by default it would simply crash the server process, which is a very bad experience for both the end-user and the website maintainer.
We need to keep this as a sync function, and either use `.then`/`.catch`, and/or use an async IIFE.